API.md

```diff
@@ -1,12 +1,120 @@
 # Authmosphere API Documentation
 
+## Token Cache
+
+Class to request and cache tokens on client-side.
+
+#### Usage
+
+```typescript
+
+// TokenCacheOptions
+const cacheOptions = {
```

Comment:
Would calling this var `options` make things a little bit less confusing? ^^

Reply:
✅